src/main/java/move/JolMoveBehavior.java

```diff
@@ -0,0 +1,44 @@
+package move;
+
+import java.util.List;
+import java.util.Map;
+import piece.Position;
+import piece.Route;
+import piece.Team;
+
+public class JolMoveBehavior implements MoveBehavior {
+
+    private static final Map<Team, List<Directions>> teamCanMoveDirection;
+
+    static {
+        final List<Directions> blueCanMoveDirections = List.of(
+                new Directions(
+                        List.of(Direction.RIGHT)
+                ),
+                new Directions(
+                        List.of(Direction.LEFT)
+                ),
+                new Directions(
+                        List.of(Direction.DOWN)
+                )
```

Comment:
```suggestion
                new Directions(List.of(Direction.RIGHT)),
                new Directions(List.of(Direction.LEFT)),
                new Directions(List.of(Direction.DOWN))
```
불필요한 공백은 제거되는 것이 좋겠네요